tests/installation/await_install.pm

```diff
@@ -125,6 +125,13 @@ sub run {
     }
     # on s390 we might need to install additional packages depending on the installation method
     if (check_var('ARCH', 's390x')) {
+      # For poo#93949, we need password possibility
+      if (!is_sle && !is_leap) {
```

Comment:
what about `is_tumbleweed` ?

Reply:
This if case is Tumbleweed and all what is based on new versions.

Reply:
I think with `is_sle_micro` this is actually no longer identical to `is_tumbleweed`...

Reply:
So I should add _!is_sle_micro_?

Reply:
That is added